src/components/ReportActionItem/MoneyRequestReportPreview/MoneyRequestReportPreviewContent.tsx

```diff
@@ -427,244 +429,241 @@ function MoneyRequestReportPreviewContent({
+                                        }}
+                                        isDisabled={isOffline && !canAllowSettlement}
+                                        isLoading={!isOffline && !canAllowSettlement}
+                                    />
+                                )}
+                                {!!shouldShowExportIntegrationButton && !shouldShowSettlementButton && !shouldShowRBR && (
+                                    <ExportWithDropdownMenu
+                                        policy={policy}
+                                        report={iouReport}
+                                        connectionName={connectedIntegration}
+                                        wrapperStyle={[buttonMaxWidth, styles.flexReset]}
+                                        dropdownAnchorAlignment={{
+                                            horizontal: CONST.MODAL.ANCHOR_ORIGIN_HORIZONTAL.RIGHT,
+                                            vertical: CONST.MODAL.ANCHOR_ORIGIN_VERTICAL.BOTTOM,
+                                        }}
+                                    />
+                                )}
+                                {!shouldShowSubmitButton && shouldShowRBR && (shouldShowSettlementButton || !!shouldShowExportIntegrationButton) && (
+                                    <Button
+                                        icon={Expensicons.DotIndicator}
+                                        iconFill={theme.danger}
+                                        iconHoverFill={theme.danger}
+                                        text={translate('common.review', {amount: shouldShowSettlementButton ? getSettlementAmount() : ''})}
+                                        onPress={() => {}}
+                                        style={buttonMaxWidth}
+                                    />
+                                )}
+                                {shouldShowSubmitButton && (
+                                    <Button
+                                        success={isWaitingForSubmissionFromCurrentUser}
+                                        text={translate('iou.submitAmount', {amount: getSettlementAmount()})}
+                                        style={buttonMaxWidth}
+                                        onPress={() => iouReport && submitReport(iouReport)}
+                                        isDisabled={shouldDisableSubmitButton}
+                                    />
+                                )}
                             </View>
                         </View>
-                    </PressableWithoutFeedback>
-                </View>
-                <DelegateNoAccessModal
-                    isNoDelegateAccessMenuVisible={isNoDelegateAccessMenuVisible}
-                    onClose={() => setIsNoDelegateAccessMenuVisible(false)}
+                        {isEmpty(transactions) && <EmptyRequestReport />}
```

Comment:
`transactions` will be always an array, so there's no need to use lodash. A couple months ago we decided to keep lodash usage to a bare minimum.
```suggestion
                        {transactions.length === 0 && <EmptyRequestReport />}
```